element-template-generator/congen-cli/README.md

```diff
@@ -34,6 +34,7 @@ Run `congen -h` to see the usage information and a list of available parameters.
 `congen` supports the following commands:
 - `generate` invokes the selected generator implementation and prints the generated template to the standard output.
 - `scan` invokes the generator in a dry-run mode and prints the information
+- `operations` prints the list of all available operations both supported and unsupported
```

Comment:
From the standpoint of the CLI tool, can this be a modification of the existing `scan` operation? I think the idea behind those two is the same, and it's just the question of data formatting, but please correct me if I'm wrong

Reply:
I can totally see why it would seem like that, but they print different data to the output. As it says in the docs, the scan will actually perform a dry-run, create the template and return with the template data too. The template can only contain supported operations. For the https://github.com/camunda/product-hub/issues/2049 we need to list the unsupported operations as well. Also we need additional data, that's why we have to transform the data. I can see `scan` as a useful command, and the `operations` as something for a different purpose. Please feel free to share more details about your perspective if you think that would be a better solution.

Reply:
For history: this was resolved in DM, the core reason is maintaining abstraction isolation